webui/src/components/Swap/index.js

```diff
@@ -4,24 +4,28 @@ import { provider, contracts } from '../../environment'
 import SwapNFTsForToken from './SwapNFTsForToken'
 import RobustSwapNFTsForToken from './RobustSwapNFTsForToken'
 import SwapETHForAnyNFTs from './SwapETHForAnyNFTs'
+import SwapETHForSpecificNFTs from './SwapETHForSpecificNFTs'
 import RobustSwapETHForAnyNFTs from './RobustSwapETHForAnyNFTs'
 import SwapERC20ForAnyNFTs from './SwapERC20ForAnyNFTs'
 import SwapERC20ForSpecificNFTs from './SwapERC20ForSpecificNFTs'
-// import RobustSwapERC20ForAnyNFTs from './RobustSwapERC20ForAnyNFTs'
+import RobustSwapERC20ForAnyNFTs from './RobustSwapERC20ForAnyNFTs'
+import RobustSwapERC20ForSpecificNFTs from './RobustSwapERC20ForSpecificNFTs'
+import RobustSwapETHForSpecificNFTs from './RobustSwapETHForSpecificNFTs'
+
 
 
 
 const Types = {
     SwapNFTsForToken: "0",
     RobustSwapNFTsForToken: "1",
-    swapETHForAnyNFTs: "2",
-    // swapETHForSpecificNFTs: "3",
+    SwapETHForAnyNFTs: "2",
```

Comment:
Looks like the first letter is capitalized now, as it should. But inside `Swap` component, first line, `Types.swapETHForAnyNFTs` is still not capitalized, hence it's an undefined variable here. This needs to be capitalized as well.